docs/reference/api/docker_remote_api.md

```diff
@@ -116,6 +116,7 @@ This section lists each version from latest to oldest.  Each listing includes a
 * `GET /info` now returns `SecurityOptions` field, showing if `apparmor`, `seccomp`, or `selinux` is supported.
 * `GET /networks` now supports filtering by `label` and `driver`.
 * `POST /containers/create` now takes `MaximumIOps` and `MaximumIOBps` fields. Windows daemon only.
+* `GET /images/search` now takes a `filters` query parameter.
```

Comment:
API wise it does not deprecate anything really. Before, the *filtering* for `--stars` and `--automated` were done client side.